tests/unit/steps/test_steps_analyze.py

```diff
@@ -137,6 +139,30 @@ async def test_analysis_detect_current_cloud_os_release_same_release(apps, model
     assert result.current_cloud_os_release == "ussuri"
 
 
+@pytest.mark.asyncio
+async def test_analysis_detect_current_cloud_series_same_series(apps, model):
+    result = analyze.Analysis(
+        model=model,
+        apps_control_plane=[apps["rmq_ussuri"], apps["keystone_wallaby"], apps["cinder_ussuri"]],
+        apps_data_plane=[],
```

Comment:
I think that we should used mocked apps here, this is not related with app class at all and if that failed all tests like this will failed and that bad practice.

Reply:
We could but I don't think it is necessary because this is defined in `conftest.py`, which is meant to be shared across multiple test modules.